index.js

```diff
@@ -11,6 +11,18 @@ module.exports = {
     fragmentSource: fs.readFileSync(path.join(__dirname, 'src/fill.fragment.glsl'), 'utf8'),
     vertexSource: fs.readFileSync(path.join(__dirname, 'src/fill.vertex.glsl'), 'utf8')
   },
+  extrusion: {
+      fragmentSource: fs.readFileSync(path.join(__dirname, 'src/extrusion.fragment.glsl'), 'utf8'),
+      vertexSource: fs.readFileSync(path.join(__dirname, 'src/extrusion.vertex.glsl'), 'utf8')
+  },
+  extrusionpattern: {
+      fragmentSource: fs.readFileSync(path.join(__dirname, 'src/extrusionpattern.fragment.glsl'), 'utf8'),
+      vertexSource: fs.readFileSync(path.join(__dirname, 'src/extrusionpattern.vertex.glsl'), 'utf8')
+  },
+  extrusiontexture: {
```

Comment:
It took me a few moments to understand what `extrusiontexture` is. A name that captures the second-pass-ness of this shader would be ideal. I'll sleep on it.